packages/mdc-select/foundation.js

```diff
@@ -228,10 +229,11 @@ export default class MDCSelectFoundation extends MDCFoundation {
     this.adapter_.setMenuElStyle('display', 'block');
     const menuHeight = this.adapter_.getMenuElOffsetHeight();
     const itemOffsetTop = this.adapter_.getOffsetTopForOptionAtIndex(index);
+    const adjustedHeight = this.adapter_.getAdjustedMenuTop(index);
```

Comment:
Should the adapter method be named getAdjustedMenuHeight instead? You named the variable adjustedHeight.